x/records/keeper/callback_lsm_transfer_test.go

```diff
@@ -0,0 +1,112 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	_ "github.com/stretchr/testify/suite"
+
+	"github.com/golang/protobuf/proto" //nolint:staticcheck
+
+	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+
+	icacallbackstypes "github.com/Stride-Labs/stride/v9/x/icacallbacks/types"
+	recordskeeper "github.com/Stride-Labs/stride/v9/x/records/keeper"
+	"github.com/Stride-Labs/stride/v9/x/records/types"
+)
+
+var (
+	LSMTokenDenom = "cosmosvaloperxxx/42"
+)
+
+func (s *KeeperTestSuite) SetupLSMTransferCallback() []byte {
+	delegationAccountOwner := fmt.Sprintf("%s.%s", HostChainId, "DELEGATION")
+	s.CreateICAChannel(delegationAccountOwner)
+	delegationAddress := s.IcaAddresses[delegationAccountOwner]
+
+	// we need a valid ibc denom here or the transfer will fail
+	prefixedDenom := transfertypes.GetPrefixedDenom(transfertypes.PortID, ibctesting.FirstChannelID, LSMTokenDenom)
+	denomTrace := transfertypes.ParseDenomTrace(prefixedDenom)
+	ibcDenom := denomTrace.IBCDenom()
```

Comment:
Ok I think this answers my questions on the proto, the ibc_denom has information about the trace which is the route that the token took across channels

Reply:
yup!